eventmesh-examples/src/main/java/org/apache/eventmesh/common/ExampleConstants.java

```diff
@@ -51,4 +51,9 @@ public class ExampleConstants {
     public static final String EVENTMESH_SELECTOR_NACOS_ADDRESS = "eventmesh.selector.nacos.address";
 
     public static final String EVENTMESH_SELECTOR_TYPE = "eventmesh.selector.type";
+
+	public static final String env = "P";
+	public static final String idc = "FT";
+	public static final String subsys = "1234";
+	public static final String SERVER_PORT = "server.port";
```

Comment:
please make sure that these variables are coded in a consistent style

Reply:
changes have been made, please review again